components/TemplateGrid/TemplateGridWrapper.vue

```diff
@@ -3,93 +3,120 @@
  * See LICENSE for license details.
  */
 <template>
-    <div class="category-tree-wrapper">
-        <TemplateGridHeader
-            :style="gridStyles"
-            :columns="columns" />
+    <div class="template-grid-wrapper">
+        <slot name="gridHeader" />
         <TemplateGridContainer
             :columns="columns"
-            :rows="rowsCount"
+            :rows="rows"
             :rows-height="rowsHeight"
-            :tree-data="filteredTreeData"
-            @toggleItem="(item) => toggleItem(item)">
+            :grid-data="filteredGridData"
+            :is-dragging-enabled="isDraggingEnabled"
+            @toggleItem="(item) => toggleItem(item)"
```

Comment:
(a) => method(a);
Is not recommended way to do that. It is no need to create an anonymous function, just @click="method"